packages/ingest/s3.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
```

Comment:
This file has a large number of eslint errors.  We need to work with Jenny to get eslint integrated into her editor.

```
cumulus s3syngran_mu % eslint packages/ingest/s3.js
The react/require-extension rule is deprecated. Please use the import/extensions rule from eslint-plugin-import instead.

/Users/mhuffnagle/Source/cumulus/packages/ingest/s3.js
    9:26  error  Expected parentheses around arrow function argument                        arrow-parens
   11:3   error  Invalid JSDoc tag (preference). Replace "return" JSDoc tag with "returns"  jsdoc/check-tag-names
   11:3   error  Missing JSDoc @param "_path" declaration                                   jsdoc/require-param
   11:3   error  Missing JSDoc return description                                           valid-jsdoc
   11:3   error  Use @returns instead                                                       valid-jsdoc
   11:3   error  There must be a newline after the description of the JSDoc block           jsdoc/newline-after-description
   11:3   error  Missing JSDoc for parameter 'bucket'                                       valid-jsdoc
   11:3   error  Missing JSDoc for parameter 'key'                                          valid-jsdoc
   11:3   error  Missing JSDoc for parameter 'filename'                                     valid-jsdoc
   11:3   error  Missing JSDoc for parameter '_path'                                        valid-jsdoc
   17:11  error  'source' is assigned a value but never used                                no-unused-vars
   49:3   error  There must be a newline after the description of the JSDoc block           jsdoc/newline-after-description
   49:3   error  Missing JSDoc @param "_path" declaration                                   jsdoc/require-param
   49:3   error  Missing JSDoc for parameter 'filename'                                     valid-jsdoc
   49:3   error  Missing JSDoc return description                                           valid-jsdoc
   49:3   error  Use @returns instead                                                       valid-jsdoc
   49:3   error  Missing JSDoc for parameter '_path'                                        valid-jsdoc
   49:3   error  Invalid JSDoc tag (preference). Replace "return" JSDoc tag with "returns"  jsdoc/check-tag-names
   60:46  error  Expected to return a value at the end of arrow function                    consistent-return
   62:44  error  'err' is already declared in the upper scope                               no-shadow
   70:14  error  Missing JSDoc comment                                                      require-jsdoc
   74:13  error  'result' is never reassigned. Use 'const' instead                          prefer-const
   74:40  error  Expected parentheses around arrow function argument                        arrow-parens
   81:9   error  Assignment to function parameter 'files'                                   no-param-reassign
   83:11  error  Assignment to property of function parameter 'params'                      no-param-reassign
   84:56  error  Unexpected block statement surrounding arrow body                          arrow-body-style
   86:20  error  Expected parentheses around arrow function argument                        arrow-parens
   88:14  error  Unnecessary 'else' after 'return'                                          no-else-return
   93:3   error  Use @returns instead                                                       valid-jsdoc
   93:3   error  Invalid JSDoc tag (preference). Replace "return" JSDoc tag with "returns"  jsdoc/check-tag-names
   93:3   error  There must be a newline after the description of the JSDoc block           jsdoc/newline-after-description
   93:3   error  Missing JSDoc return description                                           valid-jsdoc
   99:9   error  'files' is never reassigned. Use 'const' instead                           prefer-const
  101:9   error  'params' is never reassigned. Use 'const' instead                          prefer-const

✖ 34 problems (34 errors, 0 warnings)
```